### PR TITLE
Fix copy&paste mistake in PublicSign preservation

### PR DESCRIPTION
**Pull request by jkotas**

### CODE REVIEW

File: src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.PreserveCompilationContext.targets [hunk 1]
```diff
@@ -32,7 +32,7 @@ Copyright (c) .NET Foundation. All rights reserved.
         <Optimize>$(Optimize)</Optimize>
         <AssemblyOriginatorKeyFile>$(AssemblyOriginatorKeyFile)</AssemblyOriginatorKeyFile>
         <DelaySign>$(DelaySign)</DelaySign>
-        <PublicSign>$(DelaySign)</PublicSign>
+        <PublicSign>$(PublicSign)</PublicSign>
```
Comment (kasperk81) [inline src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.PreserveCompilationContext.targets:35]: deleting this line would mean the same thing?

Comment (jkotas) [reply src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.PreserveCompilationContext.targets:35]: I do not know why somebody wanted to preserve `PublicSign` sign here. If you know for sure that it does not make sense to preserve `PublicSign` here, the line can be deleted.

Comment (kasperk81) [reply src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.PreserveCompilationContext.targets:35]: you are right. this is under `DependencyFileCompilerOptions`, not a regular `PropertyGroup`.

**Review by agocke — APPROVED** [review]
LGTM

**Pull request merged**